README.md

```diff
@@ -218,3 +218,10 @@ To build the docker image run
 
 ```bash
 make docker-build
+```
+
+This will build the jar and copy it into the `ws-sdk/ws/bin` folder, from where it is then added to the container image. The image will be tagged as `dp3t-docker`. 
```

Comment:
```suggestion
This will build the jar and copy it into the `ws-sdk/ws/bin` folder, from where it is then added to the container image.
The image will be tagged as `dp3t-docker`. 
```